bugzilla2gitlab/models.py

```diff
@@ -85,9 +85,14 @@ def create_labels(self, component, operating_system):
             labels.append(component_label)
 
         # Do not create a label if the OS is other. That is a meaningless label.
-        if operating_system and operating_system != "Other":
+        if conf.map_operating_system and operating_system and operating_system != "Other":
             labels.append(operating_system)
 
+        if conf.map_keywords and keywords:
+            labels += [k.title()
```

Comment:
This is perhaps slightly pedantic, but there is kind of a lot going on in this list comprehension. Could you just make it a regular loop? Also, if you could add a comment with an example of the expected format for `keywords`, it could help elucidate why `split` and `replace` are needed.